src/enums.rs

```diff
@@ -463,6 +463,44 @@ impl From<u64> for ContentEncodingType {
     }
 }
 
+/// The compression algorithm used.
+///
+/// <div class="warning">
```

Comment:
Can you just add a `# Notes` here? No need for the custom div.

Reply:
:ok: